utils/polus-python-template/{{cookiecutter.project_slug}}/src/main.py

```diff
@@ -0,0 +1,78 @@
+{% if cookiecutter.use_bfio -%}
+from bfio.bfio import BioReader, BioWriter
+import bioformats
+import javabridge as jutil
+{%- endif %}
+import argparse, logging, subprocess, time, multiprocessing
+import numpy as np
+from pathlib import Path
+
+if __name__=="__main__":
```

Comment:
What's the

Reply:
Sorry, what's the point of this for loop? Just initializing the logger? Please put the descirption of what this if statement will do before the statement starts.

Reply:
If I use comments in the CookieCutter template, they will show up in the resulting code. What I can do is provide more code comments.